pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml

```diff
@@ -401,9 +393,17 @@ package-maintainers:
     - zre
   sternenseemann:
     # also maintain upstream package
+    - cabal2nix
+    - distribution-nixpkgs
+    - hackage-db
+    - language-nix
+    - jailbreak-cabal
     - spacecookie
     - gopher-proxy
     # other packages I can help out for
+    - cabal-install
```

Comment:
It'd be nice if we could add an entry for `@NixOS/haskell`, and add packages like `cabal2nix` and `cabal-install` under it.

There is basically never a case where we'd merge the `haskell-updates` branch if packages like `cabal2nix` or `cabal-install` are broken.

Reply:
That hinges on yet another hackage2nix feature atm.